converters/toZigbee.js

```diff
@@ -1503,6 +1506,132 @@ const converters = {
             }
         },
     },
+    danfoss_mounted_mode: {
```

Comment:
I think it will be more clean to merge all converters here together, to do this change e.g. `key: ['mounted_mode_control'],` -> `key: ['mounted_mode_control', 'thermostat_orientation'], ` and in the converter do `if (key === 'mounted_mode_control'`,etc.

Reply:
I have had a closer look at that. While it makes the devices.js more readable as one tz. entry replaces a larger number, the single entry in toZigbee becomes a lot more convoluted, as a large number of `if key ===` statements are needed both in `convertGet` and `convertSet`. This is due to the fact that several of the attributes are not marked reportable and will not automatically be reported by the Thermostat unless read explicitly, so the toZigbee includes a `convertGet`

I will set it up like that for all attributes which do not require to be read to verify that they took, but will leave the others as separate entries.

Also - does it make sense to actually read these "non reportable" values during when configuring the reporting to ensure that the system knows their values ? I had not added that, but going through this proposed change made me consider this to be sensible.

Reply:
After reviewing - it is not even feasible to do what i suggested - only one of the affected parameters is marked as reportable. All the others need to be read to confirm their value.